using-babelfish/getting-started.md

```diff
@@ -80,12 +80,18 @@ time="2017-10-10T08:59:20Z" level=info msg="control server listening in /var/run
 
 #### Installing the drivers
 
-Now we need to install the driver images into the daemon, you can install the official images with the following command:
+Now we need to install the driver images into the daemon. You can install the official images with the following command:
 
 ```bash
 $ docker exec -it bblfshd bblfshctl driver install --recommended
 ```
 
+If you want to install a driver for a single language, you can do so by specifying the language and driver path to `bblfshctl`:
+
+```bash
+docker exec -it bblfshd bblfshctl driver install python docker://bblfsh/python-driver:latest
```

Comment:
```suggestion
docker exec -it bblfshd bblfshctl driver install docker://bblfsh/python-driver:latest
```
The language will be auto-detected from the image when installing (since bblfshd v2.5.3).